scale/product/configuration/product_data_file.py

```diff
@@ -82,7 +82,8 @@ def _calculate_remote_path(self, job_exe, input_file_ids):
         if job_recipe:
             recipe = job_recipe.recipe
             recipe_type_path = get_valid_filename(recipe.recipe_type.name)
-            recipe_version_path = get_valid_filename(recipe.recipe_type.version)
+            recipe_revision = RecipeTypeRevision.objects.get_revision(recipe.recipe_type.name, recipe.recipe_type.revision_num).revision_num
+            recipe_version_path = get_valid_filename('revision_%i' % recipe.recipe_type.revision_num)
```

Comment:
The paths get pretty deep already. We're probably ok without the revision info?  Or maybe add the revision info to the recipe type directory name?

Reply:
I think we're going to leave it like this for now since we were doing type/version and this just changes it to type/revision_number. Good point of discussion though. Thanks for bringing it up!